themes/bootstrap3/templates/record/hold.phtml

```diff
@@ -56,7 +56,11 @@
           </option>
         <?php endif; ?>
         <?php foreach ($this->requestGroups as $group): ?>
-          <option value="<?=$this->escapeHtmlAttr($group['id'])?>"<?=($selected == $group['id']) ? ' selected="selected"' : ''?>>
+          <option
+            value="<?=$this->escapeHtmlAttr($group['id'])?>"
+            data-locations-label="<?=$this->transEsc('pick_up_location_label_' . $group['id'], null, $this->transEsc('pick_up_location'))?>"
```

Comment:
I'm fine with defining the translation keys in Folio.ini (based on group id) instead of hard-coding in this way.  But I'm not clear where you mean to call getConfig()?  Here in the template with something like $this->ils()->getConfig() etc.?  Or do you mean something internal to the driver's getRequestGroups(), where each group returned would include its translation key?

Reply:
What I meant:
In HoldsTrait `$checkHolds` gets the Holds configuration from the driver by calling `$catalog->checkFunction()` that in turn calls the ILS driver's `getConfig()` method. So if you return the translation keys alongside other hold configuration, you can access them in HoldsTrait and pass them on to the template. I'd consider something like `$checkHolds['requestGroupPickUpLocationLabels'][$id]`. If there are only the hard-coded values, I'd leave them out of Folio.ini and just append them to the config in the `getConfig()` method.

But now that you mentioned it, extending getRequestGroups so that it would return the labels as well would be a much simpler approach. Plus it would probably work better with the "ajaxy" implementation.